prov/netdir/src/netdir_init.c

```diff
@@ -118,7 +120,7 @@ static int ofi_nd_adapter_cb(const ND2_ADAPTER_INFO *adapter, const char *name)
 	info->fabric_attr->prov_version = FI_VERSION(OFI_ND_MAJOR_VERSION, OFI_ND_MINOR_VERSION);
 
 	info->caps = OFI_ND_CAPS;
-	info->addr_format = FI_SOCKADDR;
+	info->addr_format = FI_SOCKADDR | FI_SOCKADDR_IN | FI_SOCKADDR_IN6;
```

Comment:
Really, thanks :)
Is there is possible way to specify that provider supports multiple addr_format? If only FI_SOCKADDR is specified, is it enough to be aware from application prospective that provider supports ipv4 and ipv6 both? As for me, it says that the provider supports `struct sockaddr` and I should pass instance of `struct sockaddr`.
I've checked verbs provider that supports `sockaddr_in`, `sockaddr_in6`, `sockaddr_ib`, verbs provider just specifies nothing in addr_format field :)